webapp/src/pages/attendance/components/AttendanceTable.js

```diff
@@ -0,0 +1,293 @@
+            statusCode: result.statusCode,
+            success: success
+          }
+        });
+      });
+    });
+  };
+
+  handleUndo = () => {
+    const { eventId, selectedUser } = this.state;
+    this.setState({ undoing: true }, () => {
+      attendanceService
+        .undoConfirmation(eventId, selectedUser.user_id)
+        .then(response => {
+          this.setState({
+            undoResult: {
+              undo: response.data,
+              undoError: response.error,
+              undoing: false
+            }
+          });
+        });
+    });
+  };
+
+  handleContinue = () => {
+    this.setState(
+      {
+        showDetailsModal: false,
+        loading: true,
+        confirmResult: null,
+        undoResult: null
+      },
+      () => this.getAttendanceList()
+    );
+  };
+
+  onSearchChange = field => {
+    let value = field.target.value.toLowerCase();
+    this.setState(
+      {
+        searchTerm: value
+      },
+      () => this.filterList()
+    );
+  };
+
+  filterList = () => {
+    let value = this.state.searchTerm;
+    let filteredList = this.state.originalAttendanceList.filter(
+      u =>
+        u.firstname.toLowerCase().indexOf(value) > -1 ||
+        u.lastname.toLowerCase().indexOf(value) > -1 ||
+        u.email.toLowerCase().indexOf(value) > -1
+    );
+    this.setState({ filteredList: filteredList });
+  };
+
+  render() {
+    const {
+      filteredList,
+      loading,
+      error,
+      confirming,
+      undoing,
+      confirmResult,
+      undoResult,
+      searchTerm,
+      selectedUser,
+      originalAttendanceList
+    } = this.state;
+
+    const loadingStyle = {
+      width: "3rem",
+      height: "3rem"
+    };
+
+    let confirmResultDiv = null;
+    if (confirmResult) {
+      if (confirmResult.success) {
+        confirmResultDiv = (
+          <div class="alert alert-success">
+            Success - Attendance Confirmation.
+          </div>
+        );
+      } else {
+        confirmResultDiv = (
+          <div class="alert alert-danger">
+            {" "}
+            Failure - Attendance Confirmation. Http Code:
+            {confirmResult.statusCode} Message: {confirmResult.confirmError}
+          </div>
+        );
+      }
+    }
+
+    let undoResultDiv = null;
+    if (undoResult) {
+      if (undoResult.undo && !undoResult.undoError) {
+        undoResultDiv = (
+          <div class="alert alert-success">
+            Success - Undo Attendance Confirmation.
+          </div>
+        );
+      } else {
+        undoResultDiv = (
+          <div class="alert alert-danger">
+            {" "}
+            Failure - Undo Attendance Confirmation. Message :{" "}
+            {undoResult.undoError}
+          </div>
+        );
+      }
+    }
+
+    let userInfoDiv = null;
+    if (confirmResult && confirmResult.confirmed) {
+      userInfoDiv = (
+        <div>
+          <h1>
+            {selectedUser.firstname} {selectedUser.lastname}
+          </h1>
```

Comment:
I think there's too much information here, ideally this should be short and to the point, maybe just show:
The name and t-shirt size
and add like those bootstrap badge things for accommodation award or invited guest (showing the role) and shirt size, if that information is available, otherwise just don't display anything...

Reply:
Updated this.